tests/test_config.py

```diff
@@ -56,6 +56,18 @@ def test_process_pron(
     assert config.process_pron(input_pron) == expected_pron
 
 
+@pytest.mark.parametrize(
+    "no_segment, input_pron, expected_pron",
+    [
+        (True, "lɛ̃.ɡɥis.tik", "l ɛ̃ . ɡ ɥ i . s t i k"),
+        (False, "lɛ̃.ɡɥis.tik", "lɛ̃.ɡɥis.tik"),
```

Comment:
Ah, the first case should be `False` for `no_segment`, and the second one `True` instead. And there's a typo in `"l ɛ̃ . ɡ ɥ i . s t i k"`: s should precede not follow the syllable boundary symbol.

Also, the test cases in `test_process_pron` need updating as well (all those cases need the spaces, since segmentation is the default behavior now).

Reply:
oops! my bad. thanks for catching.